nixos/tests/nginx-proxyprotocol/_.test.nix.cert.pem

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
We can generate some certificates on the fly and don't need to commit them.

Reply:
I already reused ACME machinery but it only provides for a domain and not a wildcard, so I just copied the stuff over there and run the scripts.